src/EnergyPlus/WindowManager.cc

```diff
@@ -1759,13 +1730,6 @@ namespace WindowManager {
                     tvisPhiFit(IPhi) = 0.0;
 
                     Phi = double(IPhi - 1) * 10.0;
-                    for (IGlass = 1; IGlass <= NGlass; ++IGlass) {
-                        LayPtr = Construct(ConstrNum).LayerPoint(LayerNum(IGlass));
-                        if (Material(LayPtr).GlassSpecAngTransDataPtr) {
-                            Phi = TableLookup(PerfCurve(Material(LayPtr).GlassSpecAngTransDataPtr).TableIndex).X1Var(IPhi);
-                            break;
-                        }
-                    }
```

Comment:
I assume 10 degree increment is used. I think this is not useful. As I remembered, a Belgium company wants an angle between 80 and 90, such as 85. We should not add this restriction.

Reply:
EnergyPlus generates the overall solar and visible properties of the glazing system by generating an internal table at various incident angles and then fitting a 6 degree polynomial to it. It used every 10 degrees between 0 and 90 for all other methods except the "spectral + angle" method (which used the superset of angles provided in the three tables).

It seems to me that this would be better handled by letting the user explicitly define the values of the angles used to construct the whole-system angular dependencies if they want better resolution. The way it was set implicitly based on user input into the table is potentially confusing. Furthermore, if a glazing construction had a mix of methods, the value of `TotalIPhi` is not necessarily the same for each layer and you could likely end up averaging in values that weren't intended to be used for some layers. The current calculation approach **requires** that all layers have the same number of angles defined.

Reply:
Also, I don't know if the 6 degree polynomial is a great idea either. It would probably be better to use cubic interpolation of an internally generated table.

Reply:
The 6 degree polynomial is not a perfect way to handle optical properties. Hope window engine calculation will provide more accurate way to handle.